test_scripts/Resumption/InteriorVehicleData/016_InteriorVD_resumption_two_apps_same_moduleTypes_same_moduleIds_IGN_OFF.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data for two apps after IGN_OFF in case apps are subscribed to
+--  the same moduleTypes with the same moduleIds
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app1 and app2 with REMOTE_CONTROL hmi type are registered and activated
+-- 3. App1 is subscribed to module_1
+-- 4. App2 is subscribed to module_1
+--
+-- Sequence:
+-- 1. IGN_OFF and IGN_ON are performed
+-- 2. Apps start registration with actual hashIds after SDL restart
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(module_1) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to both mobile apps
```

Comment:
b. SDL get a successful response for GetInteriorVehicleData request regarding moduleType_1 with isSubscribed=true from HMI
c. SDL respond RAI(success=true, result code = SUCCESS) to both mobile apps
d. update hashId after successful resumption

Reply:
Done